frontend/src/pages/notebookController/screens/server/NotebookServer.tsx

```diff
@@ -55,17 +57,15 @@ export const NotebookServer: React.FC = () => {
               <ActionList>
                 <ActionListItem
                   onClick={() => {
-                    if (notebook.metadata.annotations?.['opendatahub.io/link']) {
-                      window.location.href = notebook.metadata.annotations['opendatahub.io/link'];
-                    } else {
+                    if (link === '#') {
```

Comment:
I suggest you call `preventDefault()` if the link is invalid to prevent the user from ctrl clicking the link etc...
```suggestion
                    if (link === '#') {
                      e.preventDefault();
```